app/config.ts

```diff
@@ -13,6 +13,10 @@ const fallbackConfig: Config = {
   COUNTRY: 'Norway',
   MAP_CENTER: '10.7522, 63.9139',
   MAP_ZOOM: 4,
+  MAP_MAX_ZOOM: 13,
+  ZIP_DATA_ATTRIBUTION_TEXT: 'Postal code data',
+  ZIP_DATA_ATTRIBUTION_PROVIDER_NAME: 'GeoNames',
+  ZIP_DATA_ATTRIBUTION_PROVIDER_LINK: 'https://www.geonames.org/',
```

Comment:
Do we need the attribution data in the config? Can't this just be placed in the text files instead?

Reply:
(btw, the attribution that was there before is no longer relevant, so you can just place it directly there)

Reply:
I just removed the attribution as a config param and hardcoded it.